scripts/cli.py

```diff
@@ -491,6 +491,11 @@ def mcuboot():
     KnotSDK().flash_mcuboot()
 
 
+@cli.command(help='Flash Setup and Main apps')
+def flash():
+    KnotSDK().flash_signed()
+
```

Comment:
just one breaking line

Reply:
As this is a top level function declaration, there should be two blank lines between the declarations.
https://www.python.org/dev/peps/pep-0008/#blank-lines

Reply:
Ack